migration_original/ODS1Stage/tables/Base/ClientProductEntityToDisplayPartnerPhone/spu_translated_ClientProductEntityToDisplayPartnerPhone.sql

```diff
@@ -687,7 +687,7 @@ select_statement_2:= $$     with cte_swimlane as (
     --- insert Statement
     insert_statement:= ' 
                 insert(clientproductentitytodisplaypartnerphoneid,ClientProductToEntityID,DisplayPartnerCode,PhoneTypeID,PhoneNumber,SourceCode,LastUpdateDate)
```

Comment:
can you modify the identation of this please so the inserted columsn have a line each

Reply:
Ok